lib/src/signed_video_h26x_common.c

```diff
@@ -844,6 +844,23 @@ update_gop_hash(void *crypto_handle, gop_info_t *gop_info)
   return status;
 }
 
+/* hash_the_hash_list()
+ * Takes all the hash nalus from |hash_list| and hash it.
+ */
+svrc_t
+hash_the_hash_list(signed_video_t *self)
+{
+  gop_info_t *gop_info = self->gop_info;
+  uint8_t *hash = gop_info->computed_gop_hash;
+  if (gop_info->list_idx <= 0) {
+    return SV_OK;
+  }
+
+  svrc_t status =
+      openssl_hash_data(self->crypto_handle, gop_info->hash_list, gop_info->list_idx, hash);
+  return status;
```

Comment:
You can do `return openssl_hash_data(self->crypto_handle, gop_info->hash_list, gop_info->list_idx, hash);`